src/assets/main.css

```diff
@@ -31,6 +31,14 @@
   .wysiwyg a {
     @apply text-blue-600 visited:text-purple-600 underline underline-offset-2;
   }
+
```

Comment:
Leaving this in here as a reference, in case we need to test the font locally and not through a hosted url.